base/_mixins.scss

```diff
@@ -1,50 +1,93 @@
-// @file mixins.scss
-
-// @link http://csswizardry.com/2016/02/mixins-better-for-performance/
-
-// Provide an accesible hover for non-touch devices, turns it into the active state for mobile, and maintain a no-js fallback
-// @requires Modernizr as a JS dependency to get .no-touch classes
-// Adds styles via @content
-// @jhogue
-
-// Usage:
-// .link {
-//   padding: .5em 1em; // Styles that do not change when interacting with this element
-//
-//   @include touch-hover('idle') {
-//     color: black;
-//   }
-//   @include touch-hover('hover') {
-//     color: red;
-//   }
-//
-//   &__disabled {
-//     @include touch-hover('hover', true) {
-//      color: silver;
-//     }
-//   }
-// }
-
-// Outputs:
-// .link {
-//   padding: .5em 1em;
-// }
-// .link, .link:link, .link:visited {
-//   color: black;
-// }
-// .no-js .link:hover, .no-js .link:focus, .js.no-touchevents .link:hover, .js.no-touchevents .link:focus, .js.touchevents .link:active {
-//   color: red;
-// }
-// .no-js .link__disabled, .no-js .link__disabled:link, .no-js .link__disabled:visited, .no-js .link__disabled:hover, .no-js .link__disabled:focus, .js.no-touchevents .link__disabled, .js.no-touchevents .link__disabled:link, .js.no-touchevents .link__disabled:visited, .js.no-touchevents .link__disabled:hover, .js.no-touchevents .link__disabled:focus, .js.touchevents .link__disabled, .js.touchevents .link__disabled:active {
-//   cursor: default;
-//   color: silver;
-// }
-//
+////
```

Comment:
General note, this file switches between tabs and spaces. Most of the commented code are spaces, where the actual code is tabs (except when it uses spaces)